Gruntfile.js

```diff
@@ -28,12 +28,19 @@ module.exports = function (grunt) {
     watch: {
       css: {
         files: ['**/*.{scss,sass}'],
-        tasks: ['sass'],
+        tasks: ['sass', 'run:stylelint'], // run stylelint for humsci_basic/airy at same time after each change
```

Comment:
So I continued using this main or root gruntfile because having a gruntfile in each directory felt a bit tedious...